solvebio/cli/tutorial.md

```diff
@@ -19,31 +19,40 @@ View this tutorial online: https://www.solvebio.com/docs/python-tutorial
 
 ## Navigate the Library
 
-# TODO - Edit this to reflect vaults/objects instead of depositories and 
-# depository versions
-List all available depositories:
 
-    Depository.all()
+To list all available vaults, run:
 
+    Vault.all()
 
-Depositories are versioned containers of datasets. There are many versions of each depository, and each version may have one or more datasets.
 
-To list all datasets from all depositories:
+Vaults are like filesystems on a computer.  They can contain files,
+folders, and a special SolveBio-specific object called a Dataset.
+
+To list all datasets from all vaults, run:
 
     Dataset.all()
 
+To retrieve a dataset by its full path, send your account name, the vault
```

Comment:
not sure how consistent we are with this but `acme` is technically your `account domain`